modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ocp_{context}"]
+= Manually configuring {ocp} registry
+
+You can integrate {product-title} with {ocp} built-in container image registry.
+
+.Prerequisites
+* You need a *Username* and a *Password* for authentication with the {ocp} registry.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Generic Docker Registry*.
+The *Configure image integration* modal box opens.
+. Click *New Integration*.
+. Enter the required details for:
+.. *Integration Name*: The name you want to give to this integration.
+.. *Types*: Select *Registry*.
+.. *Endpoint*: The address of the registry.
+.. *Username* and *Password*.
+. Select *Test* (*`checkmark`* icon) to test that the integration with the selected registry is working.
+. Select *Create* (*`save`* icon) to create the configuration.
```

Comment:
I feel obliged to mention that ISG suggests avoiding parens when possible.  On a more productive note, is there a way to add the icons to the project so that the doc can say `Select icon::something[] to...`?  That could sidestep the clarifications that you're typing.

Reply:
```
*Create* (*`save`* icon) 
```
This format was chosen based on previous discussions. However, I like the idea of having real icons in the docs. I'll add it in my good to have list. For now I think this is fine as is.